lingua_franca/res/text/fa-ir/normalize.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
This file got deleted, so I think it's just come along for the ride and you haven't intended for it to be here.

Reply:
Done.